### PR TITLE
Added cloud URL capability constant

### PR DESCRIPTION
**Pull request by Artem-Kuz**
Users will now have the ability to add a cloudURL capability to create cloud drivers easily
#205

### CODE REVIEW

File: TestProject.OpenSDK/Drivers/TestProjectCapabilityType.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TestProjectCapabilityType"/> class.
+        /// Private constructor to prevent creating instances of sealed class.
+        /// </summary>
+        private TestProjectCapabilityType()
+        {
+        }
+
+    }
+
```
Comment (s-glatshtein) [inline TestProject.OpenSDK/Drivers/TestProjectCapabilityType.cs:39]: Please do not start & end a class with empty lines

**Comment by mstrelex** [comment]
@Artem-Kuz and @s-glatshtein 

This is a good addition, but it should be accompanied by documentation and examples:

- The README.md file needs to be updated with the relevant documentation. 
- A test should be added, same as in [Java SDK](https://github.com/testproject-io/java-opensdk/blob/master/src/test/java/io/testproject/sdk/tests/ci/drivers/CloudDriverTest.java) 
- An example demonstrating the capability usage should be added.

**Comment by Artem-Kuz** [comment]
@mstrelex I added documentation

**Review by mstrelex — DISMISSED** [review]

**Comment by mstrelex** [comment]
@Artem-Kuz 

Please clean up the commits.

**Review by mstrelex — APPROVED** [review]

**Review by mstrelex — APPROVED** [review]

**Pull request merged**